vendor/win/goobox-sync-storj.bat

```diff
@@ -0,0 +1,85 @@
+@if "%DEBUG%" == "" @echo off
+@rem ##########################################################################
+@rem
+@rem  goobox-sync-storj startup script for Windows
+@rem
+@rem ##########################################################################
+
+@rem Set local scope for the variables with windows NT shell
+if "%OS%"=="Windows_NT" setlocal
+
+set DIRNAME=%~dp0
+if "%DIRNAME%" == "" set DIRNAME=.
+set APP_BASE_NAME=%~n0
+set APP_HOME=%DIRNAME%
+
+@rem goobox-sync-storj app version
+set GOOBOX_SYNC_STORJ_VERSION=0.0.21

```

Comment:
This version number should not be hardcoded. It should be a variable/property that the build replaces with the actual version number of the release.

Reply:
I'm using [bumpversion](https://github.com/peritus/bumpversion) to manage version numbers. I think it is easier than replacing version numbers by `sed` or something like that in the CI server. 

Actually, the .bat file is made by Gradle for sync-sia. So, perhaps Maven has a similar plugin to make .bat for each release.

Reply:
I am OK moving to Gradle.